src/task/stats.ts

```diff
@@ -18,13 +20,12 @@ export default async function(task) {
   const pipeline = buildPipeline(task.config.filters, context.user['curr_project']);
 
   // stream in images from MongoDB
-  for await (const img of Image.aggregate(pipeline)) {
-
+  for await (const img of Image.aggregate<ImageSchema>(pipeline)) {
     // increment imageCount
     imageCount++;
 
     // increment reviewedCount
-    img.reviewed ? reviewed++ : notReviewed++;
+    isImageReviewed(img) ? reviewed++ : notReviewed++;
```

Comment:
we don't need to re-assess whether the image is reviewed via `isImageReviewed(img)` here, as that will already be present on the image record. Change back to:

```
    img.reviewed ? reviewed++ : notReviewed++;
```